src/qibolab/pulses.py

```diff
@@ -410,7 +410,7 @@ def fvec(t, gaussian_samples, rel_sigma, length=None):
                 return pulse
 
             num_samples = int(np.rint(self.pulse.duration * sampling_rate))
-            gaussian_samples = num_samples * (1 - self.width) // 2
+            gaussian_samples = num_samples * (1 - self.width) / 2
```

Comment:
Why moving the truncation from here to above? I don't see a clear advantage.

Reply:
Me neither, at least fully, but for now is to generate the same samples that qiskit generates as they do their calculations like this.